SU-SWS/stanford_basic
# D8CORE-3682: Fixed word wrapping on long menu titles

# READY FOR REVIEW/

# Summary
- Fixes word wrapping for very long menu items.

# Review By (Date)
- 🤷 

# Urgency
- low urgency

# Steps to Test

1. Checkout this branch locally.
2. clear your caches.
3. Create a basic page with a really long title, and give it a menu entry.  Make it a sub-menu of another menu entry.
4. Activate dropdown menus.
5. Check the dropdown to make sure the very long item wraps gracefully.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- D8CORE-3682
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
